jupyterlab/staging/index.js

```diff
@@ -149,14 +149,19 @@ function main() {
   // eslint-disable-next-line semi
   {{/each}}
 
-  var lab = window.lab = new JupyterLab({
+  var lab = new JupyterLab({
     mimeExtensions: mimeExtensions,
     disabled: disabled,
     deferred: deferred
   });
   register.forEach(function(item) { lab.registerPluginModule(item); });
   lab.start({ ignorePlugins: ignorePlugins });
 
+  // Expose global lab instance when in dev mode.
+  if ((PageConfig.getOption('devMode') || '').toLowerCase() === 'true') {
```

Comment:
Will this ever evaluate to `true`? Is the fact that there are two `index.js`s enough?

Reply:
If you start with `jupyter lab --dev-mode` that expression will evaluate to `true`.

I am not sure I understand what you mean by the two `index.js`s.

Reply:
(Basically when you have the red banner.)

Reply:
I mean the `index.js` in `jupyterlab/staging` vs the `index.js` in `dev_mode`.

Reply:
The Dev mode version is copied to staging during the release process.  They can differ before that if needed.

Reply:
Ah, got it. Thanks!